openedx/core/lib/tests/test_completion.py

```diff
@@ -0,0 +1,105 @@
+# -*- coding: utf-8 -*-
+"""
+Tests for completion utils.
```

Comment:
Same with this test, shouldn't this test be part of https://github.com/edx/completion?

Reply:
edx/completion isn't really the right place, since these tests need modulestore stuff.  I put all of the integration tests between the completion repo and edx-platform here: https://github.com/open-craft/edx-platform/tree/9a6c4033f383f4e05c3390210853ef60d40d4b6a/openedx/tests/completion_integration

Reply:
Some tests were moved to the `completion` package and others to `penedx/tests/completion_integration/test_services.py`.